docs/docs/behaviors/sensor-rotate.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Sensor Rotation
+sidebar_label: Sensor Rotation
+---
+
+## Summary
+
+The Sensor Rotation behavior sends a different keypress, depending on whether the sensor is rotated clockwise or counter-clockwise.
+
+- If rotated counter-clockwise, the first keycode is sent.
+- If rotated clockwise, the second keycode is sent.
+
+### Configuration
+
+An example of how to implement an encoder to change rgb brightness:
```

Comment:
Another minor detail in passing: suggesting capitalization of "RGB" for consistency across the docs.
```suggestion
An example implementation of an encoder that changes RGB brightness is shown below:
```